### ISSUE TITLE
pip chooses wrong version

### ISSUE THREAD

**Issue by briehm** [hoytak/treedict#12]
Hi,

I tried to install this package via pip (python 3). Unfortunately, pip chose an old version, because of the version numbering:
`Using version 0.14 (newest of versions: 0.2, 0.2.1, 0.2.2, 0.10, 0.11, 0.12, 0.13, 0.14)`

In this older version, python 3 was not supported, thus the installation yielded an error (complaining about print statement without braces..)

It would be nice if you could patch the version numbering in order to support installing your package with pip/python3 without any problems.

Cheers,

**Comment by alertedsnake** [comment]
Yeah, author screwed up naming by going from 0.14 to 0.2, not thinking about how Python does semantic versioning.  You'll have to request 0.2.2 specifically to get the latest.

To fix it one would have to just go up to 0.15.0 or maybe even a 1.0 version.  However, the project seems to be abandonware, so I wouldn't rely on this actually happening.  See also my PR #11, which has been lingering a while now :)
